### ISSUE TITLE
Button is not working inside clr-input-container

### ISSUE THREAD

**Issue by Middeaslam** [vmware/clarity#5384]
## Describe the accessibility issue

_If this is a VMware product, do not fill this out! Instead consult with the accessibility team about the issue so we can consolidate all VMware related issues and track them separately._

_Provide clear description of the accessibility issue and the impact to the user. Is this only in certain use cases? Is it confusing or broken?_

## WCAG Guideline

_What WCAG guideline is being violated here and how? You should reference a guideline from https://www.w3.org/TR/WCAG21/_

## How to reproduce

_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

## Expected behavior

_A clear and concise description of what you expected to happen._

## Versions

**App**

- Angular: [e.g. 6]
- Node: [e.g. 8.10.0]
- Clarity: [e.g. 0.12.5]

**Device:**

- Type: [e.g. MacBook]
- Screen Reader: [e.g. VoiceOver, NVDA]

For screen readers, we primarily support the following combinations, and other combinations will be considered on a case by case basis.

- NVDA + Firefox (Windows)
- VoiceOver + Safari (Apple)

## Additional notes

_Add any other notes about the problem here._

**Comment by gnomeontherun** [comment]
Hi @Middeaslam please provide details about your issue so we can help, and there is no intended support for buttons inside of clr-input-container (its meant for inputs), so details about what you are trying to accomplish would be helpful.

**Comment by bdryanovski** [comment]
I'm closing this issue because no information is provided and it's inactive for 21 days!
